cmd/podman/images/save.go

```diff
@@ -18,12 +18,12 @@ import (
 )
 
 var (
-	validFormats    = []string{define.OCIManifestDir, define.OCIArchive, define.V2s2ManifestDir, define.V2s2Archive}
+	validFormats    = []string{define.OCIManifestDir, define.OCIArchive, define.V2s2ManifestDir, define.V2s2Archive, "docker-dir", "docker-archive"}
```

Comment:
The previous values are constants; why not maintain that, and copy&paste typo-prone literals for the new ones?